### PR TITLE
Pm 11 create user login page

### PR DESCRIPTION
**Pull request by xHoudek**
This only deals with user login. This does not handle shelter login or user/shelter account creation. That will come later. Here is what this does:

- User access a protected route like "/favorites"
- User gets redirected to login page (LoginPage.js)
- User enters email and password and submits form
- On submit, LoginPage.js sends a request to the API via authService.loginUser()
- API either returns an error or a JWT
- On error, LoginPage.js updates and shows error message
- On success, LoginPage.js saves JWT to localStorage and sets login state to true
- User is then redirected to the protected route they initially navigated to
- When user clicks logout button, JWT is removed from localStorage and login state is set to false

### CODE REVIEW

File: src/routes/LoginPage/LoginPage.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.loginWrapper {
+	display: flex;
```
Comment (dotte-osu) [inline src/routes/LoginPage/LoginPage.css]: Let's change it to spaces

**Review by dotte-osu — CHANGES_REQUESTED** [review]
Thank you for setting up the complicated user login! Great work! Honestly, I don't know how the code works by just looking at the code so I just checked valuable names and indentation. If you want me to look into it, could you give me some test case such as user email and password? I guess I should just run the front end with https://github.com/asching7108/purrfect-match-api/pull/7? 
If the test is not required (I trust you!), simply fix the indentation. Thanks!

**Comment by xHoudek** [comment]
To test this, you would run this branch with PM-13, create a user in the DB (can just do this directly in MySQL Workbench), then try to log in as that user. Then you can try logging out and trying to break the login. I did test it out on my own, but I'm always up for having another set of eyes on it!

**Review by dotte-osu — APPROVED** [review]
Change to spaces for real this time lol Thank you! I trust you and I will merge it for now so that we can keep going. But I am going to test some tomorrow (I want to learn it too) and if I find something, I will create a sub-task to fix it!

**Pull request merged**